CHANGELOG.md

```
@@ -32,6 +72,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/), and this
 - ESM + CJS dual output
 - CI: Node 18/20/22 matrix
 
-[Unreleased]: https://github.com/ferro-labs/ferrolabs-typescript-sdk/compare/v0.2.0...HEAD
+[Unreleased]: https://github.com/ferro-labs/ferrolabs-typescript-sdk/compare/v0.3.0...HEAD
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)